domain/consensus/consensus.go

```diff
@@ -313,7 +313,7 @@ func (s *consensus) GetVirtualInfo() (*externalapi.VirtualInfo, error) {
 	if err != nil {
 		return nil, err
 	}
-	bits, err := s.difficultyManager.RequiredDifficulty(model.VirtualBlockHash)
+	bits, err := s.difficultyManager.UpdateDAADataAndReturnDifficultyBits(model.VirtualBlockHash)
```

Comment:
Why do we update anything in a GetXXX function?

Reply:
It's the same GHOSTDAG pattern

Reply:
I think I understand what you mean, but not 100%, but forget about explaining me, this needs a lengthy explanation in the comment for UpdateDAADataAndReturnDifficultyBits, explaining why do we need to update it every time we return it? Why is it done incrementally? And what "being done incrementally" exactly means in this context.